docs/index.md

```diff
@@ -24,15 +24,15 @@ resource "scalingo_app" "my-app" {
 }
 
 # Provision a highly available PostgreSQL cluster and attach it to the application
-resource "scalingo_addon" "my-db" {
+resource "scalingo_app" "my-db" {
```

Comment:
Not sure why the doc generator has updated that resource identifier. @Soulou was this expected?

Reply:
Well, there is something wrong indeed: https://github.com/Scalingo/terraform-provider-scalingo/pull/126

I take a deeper look into that